frontend/src/scenes/surveys/surveyLogic.tsx

```diff
@@ -1235,6 +1235,23 @@ export const surveyLogic = kea<surveyLogicType>([
                 return null
             },
         ],
+        deviceTypesMatchTypeValidationError: [
```

Comment:
why not using the `surveyValidationMap` function? it was added/refactored for this use case

Reply:
`surveyValidationMap` is used to execute the validation, not validate the input value, and it's from posthog-js. or are you pointing to another thing? 🤔